Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -25,5 +25,5 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     double ampHoursUsed_;
-    BatteryData dataPoint_;
+    BatteryData previous_data_point;
```

Comment:
One more thing, the team uses `camelCase_` syntax for private variables!

Reply:
LOL I played myself